crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -611,6 +622,16 @@ impl Encryption {
         }
     }
 
+    /// Get the current verification state
+    pub fn verification_state(&self) -> VerificationState {
+        self.client.inner.verification_state.get()
+    }
+
+    /// Returns a verification state Subscriber
```

Comment:
Please add a doc example here. People usually have trouble with subscribing to async stream. Usually because some of them require pinning while others not.